catalog_tools/plots/basics.py

```diff
@@ -83,3 +85,136 @@ def plot_fmd(ax: plt.Axes, mags: np.ndarray, color: str = 'blue',
     ax.set_yscale('log')
     ax.set_xlabel('Magnitude')
     ax.set_ylabel('N')
+
+
+def plot_cum_count(
+    ax: plt.Axes,
```

Comment:
I think it would be good to make the axes optional (I can also do this- I am anyways going to push this change with the fmds)